crates/hdk/src/link.rs

```diff
@@ -122,8 +122,9 @@ pub fn delete_link(address: ActionHash, get_options: GetOptions) -> ExternResult
     })
 }
 
-/// Returns all links that reference a base hash, filtered by link type and other criteria.
-/// Use a [`GetLinksInputBuilder`] to create the [`GetLinksInput`] and optionally filter links further.
+/// Returns all links that reference a base hash, filtered by the provided [`LinkQuery`] to apply diffent filters
+/// and the [`GetStrategy`], which can be used to specify whether to look up the links locally, or fetch update our them
```

Comment:
I think something got lost in merging these sentences "fetch update our them" doesn't make sense to me?

Reply:
Actually, what I wrote didn't make sense did it. My bad!

Reply:
you're right, it should be or fetch update them